package.json

```diff
@@ -25,9 +25,10 @@
     "publish:win": "yarn run build:clean && electron-builder --win -p always",
     "icon-gen": "icon-gen -i static/Icon-Color.svg -o main/static/icons -r -m ico,icns",
     "lint": "eslint --ignore-path .gitignore .",
-    "test:current": "jest TestGroupInDetail.test.js -c test/lib/unit.config.js",
+    "test:current": "jest settings.e2e.js --testRegex 'e2e.js' -c test/lib/integration.config.js",
```

Comment:
I understand. I will be removing `yarn run test:current` before making a push from here on to avoid these conflicts. The command doesn't belong in the merged code anyway!